src/common.h

```diff
@@ -27,6 +27,21 @@
 
 #define MASK(n) (~((~0U << (n))))
 
+/*
```

Comment:
Introduce a portable and efficient implementation of clz. See https://github.com/sysprog21/mado/blob/main/include/twin_private.h#L589-L620